backend/oasst_backend/api/v1/messages.py

```diff
@@ -325,7 +325,19 @@ def mark_message_deleted(
     pr.mark_messages_deleted(message_id)
 
 
-@router.post("/{message_id}/emoji", response_model=protocol.Message)
+@router.put("/{message_id}/reintroduce", status_code=HTTP_202_ACCEPTED)
+def reintroduce_message(
+    *,
+    message_id: UUID,
+    frontend_user: deps.FrontendUserId = Depends(deps.get_frontend_user_id),
+    api_client: ApiClient = Depends(deps.get_api_client),
+    db: Session = Depends(deps.get_db),
+):
+    pr = PromptRepository(db, api_client, frontend_user=frontend_user)
+    pr.reintroduce_deleted_message(message_id)
```

Comment:
The recursive param should be exposed in the api vi a query param (maybe default true).

Reply:
Will totally do that then !